File: 26_Remove_Duplicates_from_Sorted_Array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        if not nums: return 0
        
        count = 1
        pre = nums[0]
        for i in range(1, len(nums)):
            if nums[i] != pre:
                nums[count] = nums[i]
                count += 1
        return count

File: 26_Remove_Duplicates_from_Sorted_Array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    if (!nums.length) return 0;

    let uniqueIdx = 0;
    for (let i=1; i<nums.length; i++) {
        if (nums[i] > nums[uniqueIdx]) {
            nums[uniqueIdx + 1] = nums[i];
            uniqueIdx++;
        }
    }
    return uniqueIdx + 1
};
